src/gui/attributetable/qgsattributetablemodel.cpp

```diff
@@ -42,9 +42,7 @@
 #include <limits>
 
 QgsAttributeTableModel::QgsAttributeTableModel( QgsVectorLayerCache *layerCache, QObject *parent )
-  : QAbstractTableModel( parent )
-  , mLayer( layerCache->layer() )
-  , mLayerCache( layerCache )
+  : QAbstractTableModel( parent ), mLayer( layerCache->layer() ), mLayerCache( layerCache )
```

Comment:
```suggestion
  : QAbstractTableModel( parent )
  , mLayer( layerCache->layer() )
  , mLayerCache( layerCache )
```

Reply:
The formatting updates through this PR are rather annoying...

Reply:
> The formatting updates through this PR are rather annoying...

I've redone the PR, no more formatting mess.